src/cogent3/app/__init__.py

```diff
@@ -15,20 +15,21 @@
 
 def _get_app_attr(name):
     """
-    returns app details for display
-    NB: this loads the module the app is in
+    This function returns app details for display.
+
+    Notes
+    -----
+    This function also loads the module the app is in.
     """
 
     obj = apps()[name].plugin
 
-    is_composable = hasattr(obj, "app_type") and obj.app_type != App_type.NON_COMPOSABLE
-
     _types = _make_types(obj)
 
     return [
         obj.__module__,
         name,
-        is_composable,
+        hasattr(obj, "app_type"),
```

Comment:
Oh dear, I'm about to contradict myself! So sorry, I now see where you were applying that call to validate the app is composable. You were right!!

I think the issue I was concerned about relates to line 25. At this point we have a putative app .. some developer indicates this is a cogent3 app. I think we should introduce a check on line 26 as
```python
if not is_app(app):
    warnings.warn(f"{obj!r} from {obj.__module__!r} is not a valid cogent3 app, not skipping")
```
that is, we warn the user that app is invalid, we do not include it, but we shouldn't fail. What do you think?

Reply:
Yes - if an object return from stevedore is not is_app, it shouldn't appear in available_apps.  

I think we still need is_composable in the 3rd row of the attr list.  Otherwise the third column will be meaningless in available_apps (all will be True)

Reply:
I can't test this adequately without a mechanism to on-the-fly create entry_points that stevedore will load.

Reply:
Working on a strategy to do that by creating a temp dir, creating a setup.py and a module, pip installing it, running a test, then uninstalling it, forcing refresh of the ExtensionManager, and then deleting the temp dir.